apps/storybook/src/reusejsTheme.js

```diff
@@ -82,6 +82,18 @@ const variants = {
         font: 'cursor-pointer',
       },
     },
+    textAreaBaseClasses: {
+      wrapper: {
+        background: 'bg-gray-700 outline-none	',
```

Comment:
@Subbu-betalectic There are some extra spaces, please remove them.

Reply:
For className use the following template as an idea:

```
        <textarea
          rows={4}
          name="comment"
          id="comment"
          className="shadow-sm focus:ring-blue-500 focus:border-blue-500 block w-full sm:text-sm border-gray-300 rounded-md"
          defaultValue={''}
        />
```
It should look something like this:

![Screenshot 2022-07-29 at 10 17 10 AM](https://user-images.githubusercontent.com/23579455/181684722-ba4dcc94-22a8-4509-ac6d-de6b62a4e2b0.png)